### ISSUE TITLE
Include other patches and then release?

### ISSUE THREAD

**Issue by mvhconsult** [schinken/SimpleTimer#2]
I actively use this library, would be nice to formalize it a bit.

Examples of people adding the code to Github:
https://github.com/infomaniac50/SimpleTimer (with some patches, and forked a lot)
https://github.com/hamityanik/SimpleTimer (Fork of the above with most patches it seems)
https://github.com/schinken/SimpleTimer (with additional option for expressions)
https://github.com/kadu/SimpleTimer (?)
https://github.com/MeULEDs/SimpleTimer (?)
https://github.com/Timendainum/SimpleTimer (other version?)

**Comment by schinken** [comment]
What do you suggest? Merge the patchs from infomaniac50 and hamityanik, or merge my std::function patch to infomaniac50?

The important part is: Adding the active fork to the library manager

I once contacted the original author of this library, without success

**Comment by mvhconsult** [comment]
Yes, I am basically suggesting to either nag infomaniac50 into serving as the main maintainer, or do it here (if you don't mind). I can put work into merging some stuff, as long as we are two doing the testing.
